pyvodb/cli/videometadata.py

```diff
@@ -0,0 +1,71 @@
+import os.path
+from collections import OrderedDict
+import yaml
+import click
+
+from slugify import slugify
+from pyvodb.cli.top import cli
+from pyvodb.cli import cliutil
+
+
+yaml.SafeDumper.add_representer(OrderedDict,
+    lambda dumper, value: dumper.represent_dict(value.items()))
```

Comment:
Actually, pyvodb already solved this problem!
Use `yaml_dump` from `pyvodb.dumpers` instead of `yaml.safe_dump` :)